PhysicsTools/NanoAOD/python/nanogenDQM_cff.py

```diff
@@ -0,0 +1,31 @@
+import FWCore.ParameterSet.Config as cms
+import copy
+
+from PhysicsTools.NanoAOD.nanoDQM_cfi import nanoDQM
+from PhysicsTools.NanoAOD.nanoDQM_tools_cff import *
+from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
+
+nanogenDQM = DQMEDAnalyzer("NanoAODDQM",
+    vplots = cms.PSet(GenDressedLepton = nanoDQM.vplots.GenDressedLepton,
+        GenIsolatedPhoton = nanoDQM.vplots.GenIsolatedPhoton,
+        GenJet = nanoDQM.vplots.GenJet,
+        GenJetAK8 = nanoDQM.vplots.GenJetAK8,
+        GenMET = nanoDQM.vplots.GenMET,
+        GenPart = nanoDQM.vplots.GenPart, 
+        GenVisTau = nanoDQM.vplots.GenVisTau,
+        LHEPart = nanoDQM.vplots.LHEPart,
+        LHEScaleWeight = nanoDQM.vplots.LHEScaleWeight,
+        LHEPdfWeight = nanoDQM.vplots.LHEPdfWeight,
+        PSWeight = nanoDQM.vplots.PSWeight,
+    )
+)
+
```

Comment:
in master you were importing 
from DQMServices.Core.DQMQualityTester import DQMQualityTester
any reason why not doing this here ?

Reply:
Seems that this refactor happened after 10_6_X, so I reverted back to the older setup here. Note that this follows nanoDQM.py in 10_6_X.